packages/google_maps_flutter/google_maps_flutter/example/web/index.html

```diff
@@ -38,6 +38,7 @@
 
   <!-- This API key comes from: go/flutter-maps-web-tests-api-key (GCP project: flutter-infra) -->
   <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAa9cRBkhuxGq3Xw3HPz8SPwaVOhRmm7kk&libraries=geometry"></script>
+  <script src="https://unpkg.com/@googlemaps/markerclusterer@2.5.0/dist/index.min.js"></script>
```

Comment:
@ditman is probably the best person to answer that question.

Reply:
The script tag seems to be a recommended way[^1] of injecting the library, so this will have to do. unpkg.com should be familiar to some flutter web users, because for a while that was our way of using canvaskit!

As @stuartmorgan said it's not ideal because of the new network dependency, but I'd rather have this than dealing with the shrink-wrapping of the library as an asset of the package.

[^1]: https://github.com/googlemaps/js-markerclusterer?tab=readme-ov-file#install